.github/workflows/e2e.yml

```diff
@@ -6,17 +6,24 @@ jobs:
   e2e-test:
     name: Run local stack e2e test
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+        - docker_base_image: 'ubuntu:18.04'
```

Comment:
@aarontp the Docker file says ubuntu:20.04, if 18.04 is incorrect let me know and I'll update it in the GitHub actions file, otherwise let's add a comment about why this should be 18.04

Reply:
Oh yeah, good catch, that should be `20.04` AFAIK.  Let's update that here then.

Reply:
updated